iRonCub-Mk1_1/iRonCub_wbd_noFeetFTs.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+    <robot name="iRonCub-Mk1_1" build="1" portprefix="icub" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
The robot name here is an issue,

I'll check if I can replace it with `@WBD_YARP_ROBOT_NAME@`.